catboost/tutorials/model_analysis/binarized_feature_statistics_tutorial.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
The tutorial must be fully reproducible, and they must work fast (we tested on epsilon, because we wanted to be sure, it will work fast on large data).

So it would be better to use a well known open-sourced dataset for the tutorial.
You can choose user iris from sklearn for multiclassification:
from sklearn import datasets
iris = datasets.load_iris()
Titanic for one-hot encoding:
from catboost.datasets import titanic
And
boston = datasets.load_boston() for regression on numeric features.

Reply:
Docs:

Gets statistics for the feature using the model, dataset and target.

For float features takes model's borders and computes
1) Mean target value across every bin;
2) Mean model prediction across every bin;
3) The number of objects in dataset which fall into each bin;
4) Predictions on varying feature. For every object, varies the feature value so
that it equals borders[0], borders[1], ... and counts model predictions. After that,
plots the average plot over all objects.

data -- numpy.array or pandas.DataFrame -- Data to compute statistics on -- Required parameter
target -- numpy.array or pandas.Series -- Target corresponding to data -- Required parameter
feature -- int or string -- Feature index or name in pd.DataFrame -- Required parameter
prediction_type -- str -- Prediction type used for counting mean_prediction: 'Class', 'Probability' or 'RawFormulaVal' -- None
cat_feature_values -- list or numpy.array or pandas -- Categorical feature values you need to get statistics on. Needed only for statistics of a categorical feature -- None
plot -- bool -- Plot statistics -- False

Type of return value
Python dict with binarized feature statistics. For float feature, includes
                'borders' -- borders for the specified feature in model
                'binarized_feature' -- numbers of bins where feature values fall
                'mean_target' -- mean value of target over each bin
                'mean_prediction' -- mean value of model prediction over each bin
                'objects_per_bin' -- number of objects per bin
                'predictions_on_varying_feature' -- averaged over dataset predictions for
                varying feature (see above)
            For one-hot feature, returns the same, but with 'cat_feature_values' instead of 'borders'

Reply:
Usage example:


import catboost
import pandas as pd
from catboost.datasets import titanic
from plotly.offline import init_notebook_mode
init_notebook_mode(connected=True)

train, test = titanic()
target = train.Survived
train.drop(['PassengerId', 'Survived', 'Name', 'Parch', 'Ticket', 'Cabin', 'Embarked'], 
            axis=1, inplace=True)

model = catboost.CatBoostClassifier(
    iterations=200, 
    cat_features=['Pclass', 'Sex', 'SibSp'], 
    one_hot_max_size=300)
model.fit(train, target, silent=True)

feature = 'Fare'
res = model.get_binarized_statistics(train, target, feature, plot=True)

Reply:
![image](https://user-images.githubusercontent.com/33398508/58025921-23b7e380-7b1e-11e9-82a6-5bab3e1e5e74.png)